app/controllers/posts_controller.rb

```diff
@@ -14,14 +18,12 @@ def new
 
   def create
     @post = Post.new(post_params)
-    @posts = current_user.posts
     @post.user = current_user
```

Comment:
`[Optional]` you can combine these lines using the `build` method generated by the relationship between users and posts.